netci.groovy

```diff
@@ -52,4 +52,20 @@ def project = 'dotnet/corefxlab'
         // See the documentation for this function to see additional optional parameters.
         Utilities.simpleInnerLoopJobSetup(newJob, project, isPR, "Windows ${configuration}")
     }
-}
\ No newline at end of file
+}
+
+[false].each { isPR -> // We only want to generate nuget packages for official builds
+    def nugetJobName = Utilities.getFullJobName(project, 'Debug', false)
```

Comment:
Is this the same as the previous comment? I'm not sure which line(s) these comments are referring to...do you mean that the line to the new function should be:

```false isPR -> { // We only want to generate nuget packages for official builds```